exasol_transformers_extension/deployment/scripts_deployer_cli.py

```diff
@@ -1,33 +1,68 @@
-import os
+from __future__ import annotations
+
 import click
-from exasol_transformers_extension.deployment import deployment_utils as utils
+from exasol.python_extension_common.deployment.language_container_deployer_cli import (
+    SecretParams, SECRET_DISPLAY, secret_callback)
+
 from exasol_transformers_extension.deployment.scripts_deployer import \
     ScriptsDeployer
 
 
 @click.command(name="scripts")
-@click.option('--dsn', type=str, required=True)
-@click.option('--db-user', type=str, required=True)
-@click.option('--db-pass', prompt='db password', hide_input=True,
-              default=lambda: os.environ.get(
-                  utils.DB_PASSWORD_ENVIRONMENT_VARIABLE, ""))
-@click.option('--schema', type=str, required=True)
-@click.option('--language-alias', type=str, default="PYTHON3_TE")
+@click.option('--dsn', type=str)
+@click.option('--db-user', type=str)
+@click.option(f'--{SecretParams.DB_PASSWORD.value}', type=str,
+              prompt='DB password', prompt_required=False,
+              hide_input=True, default=SECRET_DISPLAY, callback=secret_callback)
+@click.option('--saas-url', type=str,
+              default='https://cloud.exasol.com')
+@click.option(f'--{SecretParams.SAAS_ACCOUNT_ID.value}', type=str,
+              prompt='SaaS account id', prompt_required=False,
+              hide_input=True, default=SECRET_DISPLAY, callback=secret_callback)
+@click.option(f'--{SecretParams.SAAS_DATABASE_ID.value}', type=str,
+              prompt='SaaS database id', prompt_required=False,
+              hide_input=True, default=SECRET_DISPLAY, callback=secret_callback)
+@click.option('--saas-database-name', type=str)
+@click.option(f'--{SecretParams.SAAS_TOKEN.value}', type=str,
+              prompt='SaaS token', prompt_required=False,
+              hide_input=True, default=SECRET_DISPLAY, callback=secret_callback)
```

Comment:
I think, I have a déjà vu. I think, these options we can also extract at some point into the python extension common